job-runner/main.go

```diff
@@ -14,9 +14,21 @@ import (
 )
 
 func main() {
+	// Make a chain repo.
+	chainRepo := chain.NewMemoryRepo()
```

Comment:
Not using Redis repo? Or, how do we let user configure this? As OSS we can't assume user has/wants Redis, but then there should be some switch between memory-backed repo and Redis-backed repo if Redis is available.